cronos/chain.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cronoschain",
```

Comment:
```suggestion
  "chain_name": "cronos",
```

Reply:
If you prefer it to remain cronoschain, can you rename the folder to match please